model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -148,6 +148,19 @@ object WorkspaceVersions {
     )
 }
 
+case class GcpWorkspaceDeletionContext(bucketName: Option[String])
```

Comment:
`bucketName` can be unwrapped from the `Option`. All GCP workspaces will have buckets, and Azure workspace deletion results won't have a `GcpWorkspaceDeletionContext`.